Makefile

```
@@ -580,7 +582,7 @@ ci.e2e.build.%:
 e2e.build.centos-7: centos7 docker.clean-latest-ami
 
 # Run os-packages-artifacts outside devkit container.
-e2e.build.centos-7-offline: centos7-offline docker.clean-latest-ami
+e2e.build.centos-7-offline: centos7-offline docker.clean-latest-ami infra.aws.destroy
```

```suggestion
e2e.build.centos-7-offline: centos7-offline
```

What do you think about adding new steps in TC to do this cleanup `docker.clean-latest-ami infra.aws.destroy` to not leak AMIs and infra on failed tests"

---

the destroy infra step is already doing in TC. I think its worth keeping this in both in case you want to run locally